bridges/relays/lib-substrate-relay/src/messages/mod.rs

```diff
@@ -393,10 +406,21 @@ where
 	R::BridgedChain:
 		bp_runtime::Chain<AccountId = AccountIdOf<P::SourceChain>, Hash = HashOf<P::SourceChain>>,
 	CallOf<P::TargetChain>: From<BridgeMessagesCall<R, I>> + GetDispatchInfo,
+	Box<
+		bp_messages::target_chain::FromBridgedChainMessagesProof<
+			<<P as SubstrateMessageLane>::SourceChain as ChainBase>::Hash,
+			<R as pallet_bridge_messages::Config<I>>::LaneId,
+		>,
+	>: From<
+		bp_messages::target_chain::FromBridgedChainMessagesProof<
+			<<P as SubstrateMessageLane>::SourceChain as ChainBase>::Hash,
+			<P as SubstrateMessageLane>::LaneId,
+		>,
+	>,
```

Comment:
that Box stuff is exactly the suggestion from a rust compiler,
I tried this change but it does not work:

![image](https://github.com/user-attachments/assets/94a7fe41-8edd-402e-9ed7-bd7aad30e25d)

Reply:
> It can be removed if we do:
> 
> ```
> 	R: BridgeMessagesConfig<I, LaneId = P::LaneId>,
> ```
@serban300 aha, yes, thanks, fixed here: https://github.com/paritytech/polkadot-sdk/pull/5649/commits/4daae2163b645a46f763d8e6a418a63dda31b813